c2corg_ui/templates/route/view.html

```diff
@@ -162,7 +162,7 @@ <h3 class="heading show-phone"><span translate>description</span></h3>
             <div class="add-outing">
               <p class="text-center"><strong translate>no outings yet?</strong><p>
                 <button type="button" class="btn gray-btn" protected-url-btn
-                        url="'${request.route_url('outings_add')}' + '?routes=${route_id}'"
+                        url="${request.route_url('outings_add', _query={'routes': route_id})}"
```

Comment:
Is the ``routes`` param actually accepted?
In the advanced search I think we rather use ``r``. If possible it would be cool to use the same param name for consistency reasons.